app/config/tstvhq/events.json

```diff
@@ -1,84 +1,56 @@
-            "end": "2020-08-21T02:00:00-04:00",
+            "room": "grandBallroom",
+            "start": "2020-10-10T00:00:00-04:00",
+            "end": "2020-10-10T02:00:00-04:00",
             "image": "https://tstvhq.s3.amazonaws.com/events/bats.jpg"
         },
         {
             "name": "Bats in the Belfry",
             "description": "Meet your new best friend, a fruit bat or insect bat from BatWorld, a shelter in Texas that cares for these sweethearts. Learn more about their work at batworld.org",
             "room": "belfry",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-21T01:00:00-04:00",
+            "start": "2020-10-10T00:00:00-04:00",
+            "end": "2020-08-21T01:04:00-04:00",
             "image": "https://tstvhq.s3.amazonaws.com/events/bats.jpg"
         },
-        {
-            "name": "Sleeping Porch",
-            "description": "Lounge among slumbering cats on the sleeping porch, live from Best Friends Animal Society. Learn more about these cats and how you can adopt one at BestFriends.org.",
-            "room": "sleepingPorch",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-21T01:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/events/cats.jpg"
-        },
-        {
-            "name": "Parrots in the Aviary",
-            "description": "Get to know some amazing parrots in the Aviary, broadcasting live from Best Friends Animal Society. Learn how to adopt one at BestFriends.org.",
-            "room": "aviary",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-21T01:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/events/parrots.jpg"
-        },
-        {
-            "name": "Den Pups",
-            "description": "Puppies are afoot in The Den, live from Best Friends Animal Society. Learn how to adopt one at BestFriends.org.",
-            "room": "den",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-21T01:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/events/dogs.jpg"
-        },
         {
             "name": "Library",
             "description": "The Satanic Panic Archive Library is open all night. Many of these texts exist nowhere else in the world.",
             "room": "library",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-21T01:00:00-04:00",
+            "start": "2020-10-10T00:00:00-04:00",
+            "end": "2020-10-10T04:00:00-04:00",
             "image": "https://tstvhq.s3.amazonaws.com/events/library.jpg"
-        }
+        },
+         {
+            "name": "Down the Rabbit Hole",
+            "description": "Down the rabbit hole...meet Winston and Winona, Ada's beloved rabbits!",
+            "room": "den",
+            "start": "2020-10-10T00:00:00-04:00",
+            "end": "2020-08-21T01:04:00-04:00"
     ]
```

Comment:
Text highlighted in red indicates that the formatting of this file has been broken -- the section above this bracket is missing a closing `}`